src/generators/static/samples.ts.hbs

```diff
@@ -8,8 +8,10 @@
  * x-ms-original-file: {{originalFileLocation}}
  */
 import {
-{{#if hasBody }}
-  {{bodySchemaName}},
```

Comment:
If we don't use these two columns does that mean we could delete them in model and transform.ts?